make/release.mk

```diff
@@ -3,7 +3,7 @@ S3_PATH ?= "dkp/$(GIT_TAG)"
 S3_ACL ?= "bucket-owner-full-control"
 
 .PHONY: release
-release: ARCHIVE_NAME = kommander-applications_$(GIT_TAG).tar.gz
+release: ARCHIVE_NAME = kommander-applications-charts-bundle-$(GIT_TAG).tar.gz
```

Comment:
🤔 this isn’t necessarily what we consider a “chart bundle” though right? This is the git repo tar or what the docs calls the “kommander application definitions”

Reply:
🤦🏻‍♀️  I got confused again lol. We don't publish kapps charts bundle, that is true (..lemme update that message) what should we name the gitrepos 🤔  we'll want to name dkp-catalog-applications the same too

Reply:
I'll close this for now, we can open up a new pr if we want to rename